website/src/content/api-reference/commercetools-frontend-application-components.mdx

```diff
@@ -79,7 +79,7 @@ const ChannelsInfo = () => {
 | `onClose`           | `function` |    ✅    | -                 | -       | Called when the dialog closes: click on overlay, click on close button, press `ESC`.                                                                                                                                                                                                                              |
 | `children`          | `node`     |    ✅    | -                 | -       | Content rendered within the dialog. If the content is long in height (depending on the screen size) a scrollbar will appear.                                                                                                                                                                                    |
 | `size`              | `string`   |    -     | `m`, `l`, `scale` | `l`     | Horizontal width of the dialog card. If `scale` is used, the dialog size expands to the viewport size, with some margin.                                                                                                                                                                                                                                                                            |
-| `zIndex`            | `number`   |    -     | -                 | `1000`  | The `z-index` value to be applied to the overlay container. It is useful if you have stacking modals.                                                                                                                                                                                                           |
+| `zIndex`            | `number`   |    -     | -                 | -  | The `z-index` value to be applied to the overlay container. By default the all modal containers are automatically assigned a correct `z-index` value. If this prop is defined, it will overwrite the assigned value.                                                                                                                                                                                                           |
```

Comment:
Really cool work to follow 🙌🏾. Just a tiny observation here and some other repeated places. 
```suggestion
| `zIndex`            | `number`   |    -     | -                 | -  | The `z-index` value to be applied to the overlay container. By default, all the modal containers are automatically assigned a correct `z-index` value. If this prop is defined, it will overwrite the assigned value.                                                                                                                                                                                                           |
```